### ISSUE TITLE
Update to Angular 2.0 Release

### ISSUE THREAD

**Issue by sjmf** [sjmf/ng2-stompjs-demo#2]
This issue tracks updating this project to work with the Angular 2.0 release version.

The plan is to configure the build using [angular-cli](https://github.com/angular/angular-cli), as opposed to the current method using SystemJS.

**Time estimate**: 4-5 h

For an example of where I've already done this see my [MQTT demo](https://github.com/sjmf/ng2-mqtt-demo)

**Comment by JimiC** [comment]
You are in luck. I'm currently working on the same case as our project front-end is based on `Angular2` via `angular-cli` and we need `stompjs` to retrieve data from a `RabbitMQ` server.

I'll get back to you as soon as I have it working.

**Comment by JimiC** [comment]
**Update**
I have a fully functional code base at my fork. Last thing to do is to figure out what goes wrong with the testings.

Have a look at the [branch](https://github.com/JimiC/ng2-stompjs-demo/tree/angular-cli) and let me know if you want me to make any changes, before I submit a PR.

**P.S:** I'll probably submit a PR for the `STOMPjs` typings to the [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) repo. Once it gets accepted we can replace the hardcoded typings with the one from `@types`.

**Comment by JimiC** [comment]
**Update**
I managed to solve the testing issues.

I'm good to go for a PR.

Waiting for your review and input.

**Comment by sjmf** [comment]
Many thanks for this @JimiC - I'll clone the repo and build it tomorrow morning, and get back to you.

**Comment by JimiC** [comment]
**Update**
I have force pushed some tweaks to the `stompjs` typings and added an extra config property that serves as switcher to enable/disable client debugging displaying.

**Comment by JimiC** [comment]
**Update**
I updated the typings to much the one I'm submitting to `DefinitelyTyped`.

**Comment by JimiC** [comment]
**Update**
I have submitted a [PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/12605) on `DefinitelyTyped` for `stompjs` type definitions. Waiting for it to pass the tests and be merged.

Once that is done, I'll update the project with that typing. 

If by then you are OK with the code, let me know.

**Comment by sjmf** [comment]
Looks good! Go ahead with the PR and I'll accept it. :)

**Comment by sjmf** [comment]
Nice! Merged your PR. Thanks. :)

**Comment by JimiC** [comment]
I thank you for providing the insights on how to mix `ng2` and `stompjs`. Without your project I was lost.
